Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:latest
+# create a directory to hold the application code inside the image, aka the working directory of the application:
+RUN mkdir -p /usr/src/nuxt-app
+WORKDIR /usr/src/nuxt-app
+COPY package*.json yarn.lock ./
+RUN yarn install
+RUN npm rebuild node-sass
```

Comment:
This shouldn't need to happen if the modules are installed for linux already

Reply:
Are you referring to the npm rebuild node-sass?

Reply:
https://github.com/sass/node-sass/issues/2536 I was getting some error highly related with this. I had to use `npm rebuild node-sass` to fix it on my computer.

Reply:
I think this makes it work because it rebuilds the node modules in your host computers node folder into Linux versions. What we want is tell docker to use the node modules built into the docker image already. I explained on slack how to do this, it’s a pretty simple addition to the volumes field.

Reply:
Add a new volume:
`- /usr/src/nuxt-app/node_modules`
Under the other volume mount in the docker compose